src/main/java/com/amit/electronic/store/service/UserServiceImpl.java

```diff
@@ -0,0 +1,69 @@
+package com.amit.electronic.store.service;
+
+import com.amit.electronic.store.entity.User;
+import com.amit.electronic.store.model.GetUserRequest;
+import com.amit.electronic.store.repository.UserRepository;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@Transactional
+@AllArgsConstructor
+@Slf4j
+public class UserServiceImpl implements UserService {
+    private final UserRepository userRepository;
+
+    public User create(User user) {
+        return userRepository.save(user);
```

Comment:
Before performing any JDBC op, you need to have Preconditions check. 
Validate all params, Ensure correctness and everything.

Reply:
done